pmoves/docs/PMOVES.AI PLANS/README_DOCS_INDEX.md

```
@@ -1,6 +1,7 @@
 # PMOVES v5 • Documentation Index
-_Last updated: 2025-11-03_
+_Last updated: 2025-12-14_
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)